test/linters/accessibility/aria_label_is_well_formatted_test.rb

```diff
@@ -49,15 +54,14 @@ def test_does_not_warn_when_aria_label_is_excepted_in_config
     assert_empty @linter.offenses
   end
 
-
   def test_does_not_warn_if_aria_label_is_in_excepted_list
     @file = <<~HTML
      <button aria-label="hello" ></button>
-     <button aria-label="hello world" ></button>
+     <button aria-label="git checkout <%= some_branch %>"></button>
     HTML
-    @linter.config.exceptions = ["hello"]
+    @linter.config.exceptions = ["hello", "git checkout <%= some_branch %>"]
```

Comment:
Testing to see if more "complex" exceptions using string interpolation can be made.